tests/foreman/api/test_smartproxy.py

```diff
@@ -221,17 +238,16 @@ def test_positive_import_puppet_classes(self):
         """
         new_port = get_available_capsule_port()
         with default_url_on_new_port(9090, new_port) as url:
-            proxy = self._create_smart_proxy(url=url)
+            proxy = self._create_smart_proxy(request, url=url)
             result = proxy.import_puppetclasses()
-            self.assertEqual(
-                result['message'],
+            assert (
                 "Successfully updated environment and puppetclasses from "
-                "the on-disk puppet installation",
-            )
+                "the on-disk puppet installation"
+            ) in result['message']
 
 
 @pytest.mark.run_in_one_thread
-class SmartProxyMissingAttrTestCase(APITestCase):
+class TestSmartProxyMissingAttr:
```

Comment:
does this need to be a separate class? If not, then this entire file should become class-less.

Reply:
I can't see any functional reason, apart from keeping the tests logically separated.